### ISSUE TITLE
Use servers in conform_schema matcher

### ISSUE THREAD

**Issue by ShonRepo** [skryukov/skooma#6]
Hello, I'm using a scheme with endpoints like `/materials/:id`. But my full route is` /admin/v1/materials/:id`. When testing, I use `/admin/v1/materials/:id`, I receive a successfully completed request, but it also shows that this endpoint is not in the schema. Please tell me is it possible to somehow check the presence of servers during the test?

**Comment by skryukov** [comment]
Hey, @ShonRepo thanks for the issue!

There is nothing built in atm 😅, but I'll try to address the issue this week.

**Comment by ShonRepo** [comment]
thanks!

**Comment by ScotterC** [comment]
Seconding this proposal. I have a similar setup.

Excited to see this gem grow! Document first feels like this could be much simpler than alternatives but lots of room for a smooth experience.

**Comment by skryukov** [comment]
Hey, I just released v0.2.2 with a new `path_prefix` option to help resolve this issue. I decided it was much easier to just set this prefix manually rather than guessing which server is in use.

```ruby
# spec/rails_helper.rb

RSpec.configure do |config|
  # ...
  path_to_openapi = Rails.root.join("docs", "openapi.yml")
  # pass path_prefix option if your API is mounted under a prefix:
  config.include Skooma::RSpec[path_to_openapi, path_prefix: "/internal/api"], type: :request
end
```

Let me know if that works for you.

**Comment by ShonRepo** [comment]
Thanks!
